### DISCUSSION TITLE
how to find out each layer properties for multi-layer .psd file

### DISCUSSION THREAD

**Discussion by chgmxx** [ImageMagick/ImageMagick#5587] [Help]
Hi, I want to find out each layer properties for psd file, we want to know the postion (x,y,width,height) for every layer. Is there any command line. we want to import psd file to h5 for programming
thanks

**Comment by urban-warrior** [comment]
Try `identify -verbose image.psd`.  The position should display as part of the detailed attributes of each layer.

**Reply by chgmxx** [reply]
Thanks a lot. the output contains lots of information. is there any breif command. or else, can convert info to xml/json file. we need to display the upload psd file on h5 web canvas

**Comment by snibgo** [comment]
> is there any breif command. or else, can convert info to xml/json file.

How about:
```
magick in.psd out.json
```

**Reply by chgmxx** [reply]
thanks a lot, we also want to know each  layer image content(convert to png or jpg)  of psd and corresponding to layer position infomation(x,y,width,height), so, is there any command arguments?

**Reply by fmw42** [reply]
identify -verbose shows all the information that Imagemagick collects.  See https://imagemagick.org/script/identify.php

Perhaps exiftool or GIMP might show more?
